docker-compose/statistics/app/templates/index.html

```diff
@@ -0,0 +1,171 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.0.0/dist/css/bootstrap.min.css"
```

Comment:
Can we have all `.css` and `.js` files locally? This page doesn't need to depend on other web servers, which can go down or track the user.